models/contrib/nish/curve.py

```diff
@@ -238,7 +238,7 @@ def run(self, input: CurveDepeggingAmountInput) -> CurvePoolDepeggingAmount:
             raise ModelRunError('Pool with >2 token not implemented.')
 
         return CurvePoolDepeggingAmount(
-            pool_info = pool_info,
+            pool_info = CurvePoolPeggingInfo(**pool_info),
```

Comment:
should this be somewhere else?

Reply:
Sure. I moved this up to return the CurvePoolPeggingInfo object and not to use the dict.